open_spiel/bots/stockfish/stockfish_bot.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef OPEN_SPIEL_BOTS_STOCKFISH_STOCKFISH_BOT_H_
+#define OPEN_SPIEL_BOTS_STOCKFISH_STOCKFISH_BOT_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "open_spiel/spiel_bots.h"
+#include "open_spiel/games/chess.h"
+#include "open_spiel/bots/uci/uci_bot.h"
+
+namespace open_spiel {
+namespace stockfish {
+
+enum StockfishAnalysisContempt {
+  kBoth,
+  kOff,
+  kWhite,
+  kBlack
+};
+
+class StockfishOptionsBuilder {
+ public:
+  StockfishOptionsBuilder() { Reset(); };
+
+  StockfishOptionsBuilder *WithDebugLogFile(const std::string &debug_log_file);
```

Comment:
(not necessary) but if you feel like it, you could adapt this macro https://github.com/pytorch/pytorch/blob/22a34bcf4e5eaa348f0117c414c3dd760ec64b13/torch/csrc/api/include/torch/arg.h and add SPIEL_ARG :)